lib/scaleway/services/postgresql/main.j2.tf

```diff
@@ -0,0 +1,81 @@
+locals {
+  tags_postgresql = {
+    cluster_name = var.cluster_name
+    cluster_id = var.kubernetes_cluster_id
+    region = var.region
+    q_client_id = var.q_customer_id
+    q_environment_id = var.q_environment_id
+    q_project_id = var.q_project_id
+    database_identifier = var.postgresql_identifier
+    {% if resource_expiration_in_seconds is defined %}ttl = var.resource_expiration_in_seconds{% endif %}
+    {% if snapshot is defined and snapshot["snapshot_id"] %}meta_last_restored_from = var.snapshot_identifier{% endif %}
+  }
+  tags_postgresql_list = [for i, v in local.tags_postgresql : "${i}=${v}"] # NOTE: Scaleway doesn't support KV style tags
+}
+
+resource "scaleway_rdb_acl" "main" {
+  instance_id = scaleway_rdb_instance.postgresql_instance.id
+{%- if publicly_accessible %}
+  # By default, all IPs are authorized => 0.0.0.0/0
+  acl_rules {
+    ip = "0.0.0.0/0"
+    description = "accessible from any host"
+  }
+{%- else %}
+  # TODO(benjaminch): Allow only Scaleway's private traffic
+  acl_rules {
```

Comment:
Is it possible to change the ACL afterward ?

Reply:
this is not mentionned in their documentation, so I guess it's doable: https://developers.scaleway.com/en/products/rdb/api/#acl-rules-allowed-ips

```
ACL Rules / Allowed IPs

Access Control List Rules (ACL Rules) define which host and networks can connect to your Database Instance Endpoint. You can add, edit or delete rules from your ACLs. The initial setup of an instance allows full network access from anywhere (0.0.0.0/0).

Access control is handled directly at network-level by our load-balancers, so the Database does not need to do it itself, making the filtering more efficient, universal and relieving the Database from this task.
```